sample-app/src/components/UniversalResults.tsx

```diff
@@ -81,9 +98,14 @@ function renderVerticalSections(props: VerticalSectionsProps): JSX.Element {
         return <SectionComponent
           results={results}
           verticalKey={verticalKey}
-          header={<SectionHeader {...{ label, resultsCountConfig, appliedFiltersConfig }}/>}
+          header={<SectionHeader {...{ 
+            label, 
+            resultsCountConfig,
+            appliedFiltersConfig,
+            verticalKey,
+            viewAllButton: verticalConfig.viewAllButton 
```

Comment:
In the new mocks, is the `viewAllButton` moved to the top of a section?

Reply:
yep